### DISCUSSION TITLE
Grafana has way less graphs than it used to.

### DISCUSSION THREAD

**Discussion by DebianGuru** [Security-Onion-Solutions/securityonion#6319] [Unsupported Versions] [answered]
Just wondering what's up with Grafana.  I noticed a while ago that it (the dashboards I guess) seem to have way less info than they used to.  A few months ago I had charts for PCAP retention, ES information and all kinds of other good information.  The graphs were small and a bit crammed in, but very useful.

Seems like things are minimalistic these days with maybe 15 large graphs.  This is a standalone deployment.  Nothing has changed as far as that goes.

I thought maybe I was missing something by not being logged into it, so I tried "admin" and the password returned by "sudo salt-call pillar.get secrets", but it doesn't let me log in.  It says unknown user or password.  I don't know why not.  Could this be related?

**Comment by dougburks** [comment, marked as answer]
There should be at least one more dashboard available if you click the Dashboards icon on the left:
https://docs.securityonion.net/en/2.3/grafana.html
